docs/Configuring.md

```diff
@@ -430,28 +349,81 @@ database {
+  db {
+    driver = "org.sqlite.JDBC"
+    url = "jdbc:sqlite:file:cromwell-executions/cromwell-db/cromwell-db?foreign_keys=true&date_class=text"
+    connectionTimeout = 120000
+    numThreads = 1
+   }
+}
+
+database.metadata {
+  profile = "slick.jdbc.SQLiteProfile$"
+  db {
+    driver = "org.sqlite.JDBC"
+    url = "jdbc:sqlite:file:cromwell-executions/cromwell-db/metadata-db?foreign_keys=true&date_class=text"
+    connectionTimeout = 120000
+    numThreads = 1
+   }
+}
+```
+Explanation of the options:
 
+HSQLDB:
 * `jdbc:hsqldb:file:cromwell-executions/cromwell-db/cromwell-db;` This will make sure
    all persistence files will end up in a folder `cromwell-db` inside `cromwell-executions`.
-* `shutdown=false`. This makes sure the database will not be shutdown unless Cromwell explicitly does so.
-* `hsqlldb.default_table_type=cached`. 
-   By default hsqldb uses in memory tables, this will ensure data is written to disk and 
+* `shutdown=false` This makes sure the database will not be shutdown unless Cromwell explicitly does so.
+* `hsqldb.default_table_type=cached`
+   By default hsqldb uses in memory tables, this will ensure data is written to disk and
    decrease memory usage.
-* `hsqldb.result_max_memory_rows=10000` . Limits the amount of rows in memory for temp tables. 
-* `hsqldb.tx=mvcc` this is a  cromwell default for running with hsqldb.
-* `hsqldb.large_data=true`. Cromwell creates huge DBs that need to be opened.
-* `hsqldb.applog=1`. Log errors relating to the database.
-* `hsqldb.lob_compressed=true`. Compress lobs. This saves some space. Do note that lobs are 
+* `hsqldb.result_max_memory_rows=10000` Limits the amount of rows in memory for temp tables.
+* `hsqldb.tx=mvcc` This is a cromwell default for running with hsqldb.
+* `hsqldb.large_data=true` Cromwell creates huge DBs that need to be opened.
+* `hsqldb.applog=1` Log errors relating to the database.
+* `hsqldb.lob_compressed=true` Compress lobs. This saves some space. Do note that lobs are
   compressed individually. The total database will still contain a lot of redundancy because a
   lot of lobs will be similar.
-* `hsqldb.script_format=3`. Compress script. (uses gzip internally). 
+* `hsqldb.script_format=3` Compress script. (uses gzip internally).
    The script can still be opened normally after decompressing with gzip.
-* `connectionTimeout = 120000` opening the large database files again when running cromwell will 
+* See also http://hsqldb.org/doc/guide/dbproperties-chapt.html
+
+SQLite:
+* `jdbc:sqlite:file:cromwell-executions/cromwell-db/cromwell-db?` This will make sure
+   all persistence files will end up in a folder `cromwell-db` inside `cromwell-executions`.
+* `foreign_keys=true` Enable SQLite Foreign Key support.
+* `date_class=text` Ensure dates are stored the same way they are retrieved, as text.
+* See also https://www.sqlite.org/pragma.html
+
+Slick:
+* `connectionTimeout=120000` opening the large database files again when running cromwell will
```

Comment:
The long connection timeout is specific to the HSQLDB file database though. 3GB (!) of files is not uncommon, and that is after a single workflow run with a few hundred jobs. I wonder how big the sqlite database will end up to be.